packages/share/android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.share">
+    <application>
+      <provider
+        android:name="io.flutter.plugins.share.ShareFileProvider"
+        android:authorities="${applicationId}.flutter.share_provider"
+        android:exported="false"
+        android:grantUriPermissions="true">
+        <meta-data
+          android:name="android.support.FILE_PROVIDER_PATHS"
+          android:resource="@xml/flutter_share_file_paths"/>
+      </provider>
```

Comment:
I was wondering [merge rule markers](https://developer.android.com/studio/build/manifest-merge#merge_rule_markers) can be of use here instead of introducing another provider 🤔

Reply:
Good point. This was as well what I hated most in this plugin. But wonder what other provider to merge it with and if meta-data could make problems if merged with another. Specially if the other package as well uses FILE_PROVIDER_PATHS with a different config. Basically I know too little about how to handle this here. If anyone has more knowledge and can tell me how (or even create a PR on to update my feature branch) that would be great :)